package.json

```diff
@@ -35,6 +35,7 @@
     "del": "^3.0.0",
     "eslint-config-skyscanner-with-prettier": "^0.3.0",
     "eslint_d": "^7.1.1",
+    "fs": "0.0.1-security",
```

Comment:
This looks like it shouldn't be included

Reply:
I read the json file using fs (to parse the token names)

Reply:
`fs` is builtin to node. This is probably something completely different. Please remove it